### DISCUSSION TITLE
Temperature in motion preset when no presence?

### DISCUSSION THREAD

**Discussion by mldytech** [jmcollin78/versatile_thermostat#264] [Q&A] [answered]
Hi,

is there any way to set a temperature in motion preset when there is no presence at all? 
Right now, the motion state is configured to eco when "no motion" is detected, and to comfort if there is "motion". When nobody is home, the temperature is therefore set to eco. However, I would like to set this to a lower temperature.

Thanks and best regards,
mldy

**Comment by jmcollin78** [comment]
Hello, since today, ;you have another preset name Frost protection.
You have to use motion with preset. You cannot set temperature manually.

**Reply by mldytech** [reply]
Hello, thanks for your reply. I really appreciate it. 

Maybe I expressed myself a bit misleading and you could help me with my concrete scenario. I imagine that this could be a common scenario for others too. For Example:

In the livingroom:

-  I would like to set an Eco-Temperature (e. g. 18°C) when I'm home (presence) but **not** actively in the room (no motion)
- When I'm in the room (presence and motion) I would like to set a comfort temperature (e. g. 21°C)
- If  I'm not home at all (no motion and no presence) I would like to set a lower (away) temperature (like the ones you can set in the configuration for the other preset modes when no presence is detected)

Is there any way to implement this behavior, or is it simply not possible right now?

Thank you very much!

**Reply by jmcollin78** [reply, marked as answer]
Hello, I understand your case but this is not possible to natively use both presence and motion and combine them.
But this could be an interesting improvement and I think very simple to do.

What I think is to use the Eco preset temp if present or the Eco_away preset temp if not present.
I will keep it as an enhancement issue.

**Reply by mldytech** [reply]
Alright, thank you very much!

**Reply by jmcollin78** [reply]
https://github.com/jmcollin78/versatile_thermostat/issues/267

**Reply by gdessard** [reply]
Hello,

I would do the same and the night 
Choose presence and motion.
Motion on : boost
Motion off : confort

I do this configuration : 

Boost : 20 °C
Boost abs : 16 °C
Confort : 18 °C
Confort abs : 16 °C
Eco : 15
Eco abs : 15

If presence and motion : boost 20
If not presence and motion : Boost abs (i have animal and don’t want start) 16
If presence and not motion : confort 18
If not presence and not motion : confort abs 16 

At 22h, set eco mode (for have 15 the night)
At 7h00, set activity for have presence and motion.

After 7h
It’s work when the state change to activity, the presence or not the piece is 18 or 16. And if motion after 20. It’s perfect. But if presence is off 16 become on, the température not pass to 18 and stay to 16 (same for 18 not pass to 15)

For fix, when climate.versatile_presence_state change (and not night) , change to none, litle timer and change to activity. Then i have well 18 or 16 if presence or not. 

Best regards 
Grégoire
